staff/lab/mod-printer

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+"""Remove or add printers to/from CUPS classes."""
+import argparse
+import subprocess
+import sys
+
+
+def modify_printer(args):
+    """Perform internal call to CUPS CLI to modify class(es)."""
+    # Determine which CUPS classes are available
+    cups_classes = subprocess.check_output(["lpstat", "-p"]).decode("utf-8").split("\n")
+    # Remove empty trailing newline
+    cups_classes.remove("")
```

Comment:
Better way:

```py
subprocess.run(...).strip().split('\n')
```
In other words: strip the whitespace before splitting, then you don't have to delete it on a second line.